README.md

```diff
@@ -37,6 +38,15 @@ blink.setupSystem()
 class Blink
 ```
 
+## Constructor
+* `email` - your Blink account email
+* `password` - your Blink account password
+* `deviceId` - identifies your device and registers it in your account. It's required since version 4.0.0 of this package as this is when Blink switched to 2-factor authentication flow
```

Comment:
It is not clear from the description where the deviceId is taken from. 
Can it be freely defined?

Reply:
Yes, you define it yourself. I'll add a note about that